.codegen/cmds-account.go.tmpl

```diff
@@ -11,20 +11,21 @@ import (
 	{{.SnakeName}} "github.com/databricks/cli/cmd/account/{{(.TrimPrefix "account").KebabName}}"{{end}}{{end}}{{end}}
 )
 
-var accountCmd = &cobra.Command{
-	Use:   "account",
-	Short: `Databricks Account Commands`,
-}
-
-func init() {
-	root.RootCmd.AddCommand(accountCmd)
+func New() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "account",
+		Short: `Databricks Account Commands`,
+	}
 
 	{{range .Services}}{{if .IsAccounts}}{{if not (in $excludes .KebabName) -}}
-	accountCmd.AddCommand({{.SnakeName}}.Cmd)
+	cmd.AddCommand({{.SnakeName}}.New())
 	{{end}}{{end}}{{end}}
 
-	// Register commands with groups
-	{{range .Services}}{{if .IsAccounts}}{{if not (in $excludes .KebabName) -}}
-	{{.SnakeName}}.Cmd.GroupID = "{{ .Package.Name }}"
-	{{end}}{{end}}{{end}}
+	// Register all groups with the parent command.
+	groups := Groups()
+	for i := range groups {
+		cmd.AddGroup(&groups[i])
+	}
```

Comment:
It's different, but they're related.

As of this change, we no longer can add these groups to the command from `cmd/account/groups.go` (see https://github.com/databricks/cli/pull/595/files#diff-df954b47d74c043438a0ef20993014d723b09e4f8563aa202a514997bb32df8fL36-L42) so that functionality has moved to this function. Assigning the correct group IDs to each command now happens as part of the command generation itself (in the service template).